daemon/cmd/daemon_main.go

```diff
@@ -210,6 +210,9 @@ func InitGlobalFlags(cmd *cobra.Command, vp *viper.Viper) {
 	flags.Duration(option.ConntrackGCInterval, time.Duration(0), "Overwrite the connection-tracking garbage collection interval")
 	option.BindEnv(vp, option.ConntrackGCInterval)
 
+	flags.Duration(option.ConntrackGCMaxInterval, time.Duration(0), "Set the maximum for the automatic connection-tracking garbage collection interval calculation")
```

Comment:
Are we interpreating this flag in seconds? If so, do we want to state that in the desc here? 

If not, do we want to have an implicit or explicit 'min' value as well?